.github/workflows/release-dotnet-native.yaml

```diff
@@ -40,7 +42,10 @@ jobs:
           mkdir -p dotnet/nuget/runtimes/linux-musl-arm64/native/
           tar -xvzf libextism-aarch64-unknown-linux-musl-main.tar.gz -C dotnet/nuget/runtimes/linux-musl-arm64/native/
 
+      - name: Pack NuGet packages
+        run: |
+          find ./dotnet/nuget -type f -name "*.csproj" -exec dotnet pack {} -o release-artifacts \;
+
       - name: Publish NuGet packages
         run: |
-          dotnet pack .\dotnet\nuget\Nuget.sln -o release-artifacts
           dotnet nuget push --source https://api.nuget.org/v3/index.json ./release-artifacts/*.nupkg --api-key ${{ secrets.NUGET_API_KEY }}
```

Comment:
@chrisdickinson 
> Great work! Landing this and following up by splitting the dotnet pack creation from the dotnet nuget push steps is perfect.

Is this what you meant?

Reply:
Sorry I'm late in responding! Not quite – I meant splitting the `dotnet pack` step into a separate workflow that creates a draft GitHub release on push to main or tag creation, then modifying the release workflow that triggers on GitHub release to `dotnet nuget push` the artifact from the release. (Splitting out these steps moves us in this direction, though – the PR LGTM!)